### PR TITLE
Fix/bugs

### PR DESCRIPTION
**Pull request by isVivek99**
### Is it bug?
Yes

### \*Dev Tested?

- [x] Yes
- [ ] No

#### Platforms

- [x] Web

#### Browsers

- [x] Chrome
- [ ] Safari
- [ ] Firefox

### Before / After Change Screenshots

> For visual or interaction changes. Can be video / screenshot.

### CODE REVIEW

**Comment by harshith-venkatesh** [comment]
Please add screenshot or video testing it.
Changes look good

**Review by harshith-venkatesh — APPROVED** [review]

File: app/components/common/eventModal/index.tsx [hunk 1]
```diff
@@ -19,22 +19,21 @@ interface EventModalProps {
   events: CalEvent[];
   currentEvent: CalEvent;
   createEvent?: (event: CalEvent) => void;
-  isNewEvent: boolean;
```
Comment (iRohitGaur) [inline app/components/common/eventModal/index.tsx:22]: how are we identifying a new event now?

**Comment by iRohitGaur** [comment]
Later we will have to look into getting the access of local state in order to cut down on the server call at individual event route

File: app/components/common/eventModal/index.tsx (hunk 1 above)
Comment (isVivek99) [reply app/components/common/eventModal/index.tsx:22]: through the router state

Comment (isVivek99) [reply app/components/common/eventModal/index.tsx:22]:
```
const params = useParams();
....
if (params.eventId !== 'new') {
...
}
```

**Review by iRohitGaur — APPROVED** [review]

**Pull request merged**